docs/examples/indie-linter-v2.md

```diff
@@ -0,0 +1,77 @@
+# Indie Linter v2
+
+The Indie Linter demoed below supports [Messages v2](../linter-message-v2.md) only.
+
+## package.json
+
+Only the `consumedServices` field is important in this manifest, the rest are just for reference
+
+```json
+{
+  "name": "my-indie-linter",
+  "main": "index.js",
+  "version": "0.0.0",
+  "private": true,
+  "description": "My Linter Package",
+  "keywords": ["linter-example", "example"],
+  "engines": {
+    "atom": ">=1.4.0 <2.0.0"
+  },
+  "consumedServices": {
+    "linter-indie": {
+      "versions": {
+        "2.0.0": "consumeIndie"
+      }
+    }
+  }
+}
+```
+
+## index.js
+
+```js
+/* @flow */
```

Comment:
You are annotating this as `flow`, but it has no type annotations. I'd recommend just marking it as `'use babel';` since this is just meant to be a simple example.